python/mlcroissant/examples/imagenet/main.py

```diff
@@ -0,0 +1,771 @@
+"""ImageNet training in PyTorch.
+
+From:
```

Comment:
Do we need to copy this from pytorch into mlcroissant? Could you alternatively refer the user to https://github.com/pytorch/examples/blob/2d725b6ab255e05c55e0b08925f06f171aaedc0c/imagenet/main.py?